fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -1798,14 +1798,43 @@ Version TagPartitionedLogSystem::getPeekEnd() const {
 		return std::numeric_limits<Version>::max();
 }
 
+std::vector<Reference<LocalitySet>> TagPartitionedLogSystem::getPushLocationsForTags(
+    std::vector<int>& fromLocations) const {
+	std::vector<Reference<LocalitySet>> restrictedLogSets;
+	for (auto& log : tLogs) {
+		int locationOffset = 0;
```

Comment:
Is it correct to initialize this within the "for" loop? In the below method, it is done outside of the "for" loop.